cmd/ttn-lw-stack/commands/start.go

```diff
@@ -269,6 +270,18 @@ var startCommand = &cobra.Command{
 					Redis: redis.New(config.Redis.WithNamespace("as", "io", "webhooks")),
 				}
 			}
+			switch cfg := config.AS.DeviceCache; cfg.Backend {
+			case "memory":
+				var cache gcache.Cache
+				if cfg.Size <= 0 {
+					cache = gcache.New(-1).Build()
+				} else {
+					cache = gcache.New(cfg.Size).LFU().Build()
+				}
+				config.AS.DeviceCache.Cache = applicationserver.NewMemoryDeviceCache(cache)
+			case "":
+			default:
+			}
```

Comment:
We initialize the concrete implementations in a config package, not in the start command. See many other use cases.